package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-inappbrowser",
-  "version": "3.1.0-OS3",
+  "version": "4.0.0-OS4",
```

Comment:
@GFOutsystems please reset the OS versioning. As we update the version of the plugin, we can just have OS telling us that there are custom changes applied.